test/bdd/fixtures/wellknown/config.json

```diff
@@ -0,0 +1,34 @@
+{
+  "consortium_data": {
+    "domain": "testnet.trustbloc.local",
+    "policy": {
+      "cache": {
+        "max_age": 2419200
+      },
+      "num_queries": 2,
+      "history_hash": "SHA256",
+      "sidetree": {
+        "hash_algorithm": "SHA256",
+        "key_algorithm": "NotARealAlg2018",
+        "max_encoded_hash_length": 100,
+        "max_operation_size": 8192
+      }
+    }
+  },
+  "members_data": [
+    {
+      "domain": "stakeholder.one:8088",
+      "policy": {"cache": {"max_age": 604800}},
+      "endpoints": [
+        "https://sidetree-mock:48326/sidetree/0.0.1"
+      ],
+      "privateKeyJwk": {
```

Comment:
Would it be better to separate key material to different file and reference that file here. E.g. for two domains we would have to files that are referenced from this file. This may not be necessary or it can be added later.

Reply:
I will add it in next PR